app/Http/Controllers/PostsController.php

```diff
@@ -71,7 +71,9 @@ public function __construct(PostManager $posts, SignupRepository $signups, PostT
     public function index(Request $request)
     {
         $query = $this->newQuery(Post::class)
-            ->orderBy('created_at', 'desc');
+            ->orderBy('created_at', 'desc')
+            // by default, do not return posts hidden from the gallery
+            ->withoutTag('hide-in-gallery');
```

Comment:
Would it make sense to apply the same logic as we do for `whereVisible` here (where staff members & users can see their own hidden posts)?

Reply:
yes I like this idea better @DFurnes ! I'll update that!